client/login/wp-login/index.jsx

```diff
@@ -89,7 +103,7 @@ class Login extends React.Component {
 		}
 	};
 
-	footerLinks() {
+	footerLinks = () => {
```

Comment:
It's not about public vs private, but about `this` binded or not:
Arrow funcs have their this bound to current this, while regular methods aren't,
so it's one option to achieve that for funcs we'll have to pass further, to for example, `onClick` prop.